docs/Memcached.md

```diff
@@ -1,6 +1,10 @@
 # Memcached
 
-Set `SCCACHE_MEMCACHED` to a [Memcached](https://memcached.org/) url in format `tcp://<hostname>:<port> ...` to store the cache in a Memcached instance.
+Set `SCCACHE_MEMCACHED_ENDPOINT` to a [Memcached](https://memcached.org/) url in format `tcp://<hostname>:<port> ...` to store the cache in a Memcached instance.
+
+`SCCACHE_MEMCACHED` is a deprecated alias for `SCCACHE_MEMCACHED_ENDPOINT`.
```

Comment:
please write here why it is deprecated

Reply:
Just for variables format unifying. Lots of services uses `SCCACHE_<SERVICE>_ENDPOINT` for setting up the service endpoint to connect with.

Reply:
Do you need that note in the `Memcached.md` file?